src/codewhisperer/service/inlineCompletionService.ts

```diff
@@ -435,12 +443,20 @@ export class InlineCompletionService {
     setCodeWhispererStatusBarLoading() {
         this._isPaginationRunning = true
         this.statusBar.text = ` $(loading~spin)CodeWhisperer`
+        ;(this.statusBar as any).backgroundColor = undefined
```

Comment:
What is this ;

Reply:
it's not a typo, this line needs a explicit `;` and the formatter moved it to the next line.

Reply:
statusBar: vscode.StatusBarItem. What is the reason to cast it to any type?

Reply:
the `vscode.StatusBarItem` doesn't have a backgroundColor property, not sure why. I was referencing here: https://github.com/aws/aws-toolkit-vscode/blob/master/src/credentials/awsCredentialsStatusBarItem.ts#L73

Reply:
understood